src/Components/SmartComponents/CveDetail/CveImagesTable.js

```diff
@@ -52,6 +55,23 @@ const CveImagesTable = () => {
       apply,
       chipLabel: 'Search term',
     }),
+    checkboxFilter({
```

Comment:
This seems to be the same as the checkbox filter in ClusterImagesTable component ^. Can this be improved so that there is no code duplication?

Reply:
Good idea, I will address this in a separate PR.